lua/completion/source/lsp.lua

```diff
@@ -5,7 +5,7 @@ local match = require 'completion.matching'
 local opt = require 'completion.option'
 local M = {}
 
-M.callback = false
+M.handler = false
```

Comment:
honestly I have no idea what this one does.... I'm confused by what's happening in this file.

Reply:
It's referenced in other files to determine whether or not handlers have been registered. The diff for `signature_help` shows one use

Reply:
I'll try to clean up the ambiguity here with a commit.

Initially I was focused on hotfixing but since #268 got through I have some time to take a look.

Reply:
See line 61 of `complete.lua`. Depending on these variables, handlers will be registered for completion.